(GH-2057) Add file and line number to Bolt errors (#2294)
puppetlabs/bolt: documentation/writing_tasks.md

puts result.to_json
```

> Error results typically contain the file and line number under the `details` key of an
> error. The 'file' and 'line' keys will be merged with results unless 'file' is present in the
> `_error` hash returned from a task.

Reviewer: ```suggestion
Error results typically contain the file and line number under the `details` key of an
error. Bolt merges the 'file' and 'line' keys with results unless 'file' is present in the
`_error` hash returned from a task.
```
